### PR TITLE
[T3A3][T13-A2] Kevin Ong

### PR DESCRIPTION
**Pull request by Kevin-Ong**
Ready for review

### CODE REVIEW

File: src/seedu/addressbook/commands/CommandResult.java [hunk 1]
```diff
@@ -25,6 +25,10 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
         this.feedbackToUser = feedbackToUser;
         this.relevantPersons = relevantPersons;
     }
+    
+    public String getFeedbackToUser() {
+        return this.feedbackToUser;
+    }
```
Comment (damithc) [inline src/seedu/addressbook/commands/CommandResult.java]: Avoid accumulating unrelated changes into a single PR. 
Tip: If you are currently on `branch1` and want to create a new `branch2`, you need to switch back to `master` branch before creating `branch2`. If not, changes in `branch1` will appear in the PR created from `branch2` later.

File: src/seedu/addressbook/data/person/Block.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package seedu.addressbook.data.person;
+
+public class Block {
```
Comment (damithc) [inline src/seedu/addressbook/data/person/Block.java]: Missing header comment. All non trivial classes/methods should have javadoc format header comments.

**Comment by damithc** [comment]
Good effort @Kevin-Ong Some comments added. Please close the PR after reading comments.

**Comment by Kevin-Ong** [comment]
Acknowledged :)

**Pull request closed without merging**